Dockerfile

```diff
@@ -7,17 +7,23 @@ RUN apt-get update && apt-get install -qy \
   python-software-properties \
   software-properties-common
 RUN add-apt-repository -y ppa:snappy-dev/tools-proposed
+RUN add-apt-repository -y ppa:fgimenez/snappy-cloud-image
 
 # install dependencies
 RUN apt-get update && apt-get install -qy \
   snappy-tests-job \
   build-essential \
-  sudo subunit && \
+  sudo subunit \
+  snappy-cloud-image ubuntu-core-security-utils \
+  ca-certificates && \
   rm -rf /var/lib/apt/lists/*
 
 # make jenkins sudoer
 RUN echo "jenkins ALL=NOPASSWD: ALL" >> /etc/sudoers
 
+# make udf happy
+RUN ln -s /bin/true /usr/local/bin/udevadm
```

Comment:
Sure, udev is not available in the container, I'll add a more meaningful comment